config/environments/development.rb

```diff
@@ -38,4 +38,15 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    :storage => :s3,
+    :s3_credentials => {
+      :bucket => 'roomi-dev',
+      access_key_id:     ENV['AWS_ACCESS_KEY_ID'],
+      secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
+      # access_key_id: ENV['AKIAJX5SVNW4X3FJQIRA'],
```

Comment:
# PLEASE REMOVE THIS. SUPER DANGEROUS!!!! 

I will help you squash all the commits in this branch to make this disappear from history... do you remember the story about people scrapping github looking for this?